### PR TITLE
DOC-2566: SAML Account linking changes

### PR DESCRIPTION
**Pull request by cmilesb**
Release date: 10/1

Staging links: 
https://docs.redis.com/staging/DOC-2566/rc/security/access-control/saml-sso/#link-other-accounts
https://docs.redis.com/staging/DOC-2566/rc/changelog/october-2023/

### CODE REVIEW

**Review by joniredis — APPROVED** [review]
Looks good to me!

File: content/rc/changelog/october-2023.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+linktitle: October 2023
+description: New features, enhancements, and other changes added to Redis Enterprise Cloud during October 2023.
+highlights: SAML Account linking tokens
+weight: 76
+alwaysopen: false
+categories: ["RC"]
+aliases: []
+---
+
+## New Features
```
Comment (rrelledge) [inline content/rc/changelog/october-2023.md:12]:
```suggestion
## New features
```

File: content/rc/changelog/october-2023.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+### SAML Account linking tokens
```
Comment (rrelledge) [inline content/rc/changelog/october-2023.md:14]:
```suggestion
### SAML account linking tokens
```

File: content/rc/security/access-control/saml-sso/_index.md [hunk 3]
```diff
@@ -227,17 +227,33 @@ After you set up SAML SSO for one account, you can link other accounts you own t
...
+1. Go to **Access Management > Single Sign-On**. 
 
-1. For **Account linking**, select the checkboxes for the other accounts you want to link to SAML SSO.
+1. Select **Get token**.
 
-    {{<image filename="images/rc/access-management-saml-link-accounts.png"  alt="Link other accounts to the SAML configuration screen.">}}{{</image>}}
+    {{<image filename="images/rc/saml/popup-saml-get-token.png" alt="Get Token popup">}}{{</image>}}
 
-1. Select **Save**.
+    Select **Copy** to copy the linking token.
```
Comment (rrelledge) [inline content/rc/security/access-control/saml-sso/_index.md:238]: Maybe change this line to a numbered step or combine it with the previous step like: "Select **Get token**, then copy the token."

File: content/rc/security/access-control/saml-sso/_index.md [hunk 4]
```diff
@@ -227,17 +227,33 @@ After you set up SAML SSO for one account, you can link other accounts you own t
...
 
-1. From the **Link accounts** dialog, select **Continue** to enable SAML SSO for the selected accounts.
+1. Sign in to the account that you want to link to the SAML configuration. Go to **Access Management > Single Sign-On** and then enter the copied token into the **Join an existing SAML configuration** text box. Select the arrow to confirm.
+
+    Once you do this, the owner of the original account will receive a request to link the new account to the SAML configuration.
```
Comment (rrelledge) [inline content/rc/security/access-control/saml-sso/_index.md:242]:
```suggestion
    After you do this, the owner of the original account will receive a request to link the new account to the SAML configuration.
```

File: content/rc/security/access-control/saml-sso/_index.md [hunk 5]
```diff
@@ -227,17 +227,33 @@ After you set up SAML SSO for one account, you can link other accounts you own t
...
+
+1. Sign in with the original account and select **Access Management > Single Sign-On**. You should see the new account in the **Unlinked accounts** list.
+
+    {{<note>}}
+In order to see and interact with the Redis Cloud account in the **Unlinked accounts** list, you must be an owner of the account. If you are not an owner, the account will not be displayed in the section.
```
Comment (rrelledge) [inline content/rc/security/access-control/saml-sso/_index.md:247]:
```suggestion
To see and interact with the Redis Cloud account in the **Unlinked accounts** list, you must be an owner of the account. If you are not an owner, the account will not be displayed in the section.
```

File: content/rc/security/access-control/saml-sso/_index.md [hunk 6]
```diff
@@ -227,17 +227,33 @@ After you set up SAML SSO for one account, you can link other accounts you own t
...
+    {{</note>}}
+
+1. Select **Link account**.
+
+    {{<image filename="images/rc/saml/button-saml-link-account.png" alt="The Link Account button">}}{{</image>}}
+
+    In the popup that appears, select **Confirm conversion** to finish linking the accounts.
```
Comment (rrelledge) [inline content/rc/security/access-control/saml-sso/_index.md:254]:
```suggestion
1. In the **Convert existing users** dialog, select **Confirm conversion** to finish linking the accounts.
```

File: content/rc/security/access-control/saml-sso/_index.md [hunk 7]
```diff
@@ -227,17 +227,33 @@ After you set up SAML SSO for one account, you can link other accounts you own t
...
+1. Select **Get token**.
 
-    {{<image filename="images/rc/access-management-saml-link-accounts.png"  alt="Link other accounts to the SAML configuration screen.">}}{{</image>}}
+    {{<image filename="images/rc/saml/popup-saml-get-token.png" alt="Get Token popup">}}{{</image>}}
 
-1. Select **Save**.
+    Select **Copy** to copy the linking token.
 
-1. From the **Link accounts** dialog, select **Continue** to enable SAML SSO for the selected accounts.
+1. Sign in to the account that you want to link to the SAML configuration. Go to **Access Management > Single Sign-On** and then enter the copied token into the **Join an existing SAML configuration** text box. Select the arrow to confirm.
```
Comment (rrelledge) [inline content/rc/security/access-control/saml-sso/_index.md:240]: Maybe this could be broken up into numbered steps?

**Review by rrelledge — APPROVED** [review]
Looks good to me. I left a couple of suggestions to consider.

**Pull request merged**